src/python/pants/backend/experimental/js/BUILD

```diff
@@ -0,0 +1,4 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
While I'm okay with `js_sources` for the target type, I think the backend should be called `javascript`. You only type that one time in `pants.toml` - clarity is worth it.

Reply:
Yep - agreed, will change.